migrations/V192__Add_grb_review_discussions_table.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS system_intake_internal_grb_review_discussion_posts (
+  id UUID PRIMARY KEY NOT NULL,
+  content TEXT NOT NULL,
+  voting_role grb_reviewer_voting_role_type,
+  grb_role grb_reviewer_role_type,
```

Comment:
I need to add a constraint here still.

Reply:
Added here [2e29e38](https://github.com/CMS-Enterprise/easi-app/pull/2887/commits/2e29e38318ae8e416e7e725efd5eb01130c8cb31)